docs/topics/proc_using-server-sent-events-spring-reactive-app.adoc

```diff
@@ -0,0 +1,102 @@
+[id='using-server-sent-events-in-a-spring-reactive-application_{context}']
+= Using Server-sent events in a Spring Reactive application
+
+Create a simple service that accepts HTTP requests and returns a stream of server-sent events (SSE).
+When the client establishes a connection to the server and the streaming starts, the connection remains open. The server re-uses the connection to continuously push new events to the client.
+Canceling the request closes the connection and stops the stream, causing the client to stop receiving updates form the server.
+
+.Prerequisites
+
+* Java JDK 8 or Java JDK 11 installed
+* Maven installed
+* A Maven-based application project configured link:{link-guide-spring-boot}#configuring-your-application-to-use-spring-boot_spring-boot[configured to use Spring Boot]
+
+
+.Procedure
+
+. Add `vertx-spring-boot-starter-http` as a dependency in the `pom.xml` file of your project.
++
+.`pom.xml`
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<project>
+...
+  <dependencies>
+  ...
+include::resources/spring-boot/reactive-examples/example-sse-service-pom.xml[tags=sse-service-pom-dependencies]
+  ...
+  <dependencies>
+...
+</project>
+----
+
+. Create the main class of your application:
++
+.`SseExampleApplication.java`
+[source,java,options="nowrap",subs="attributes+"]
+----
+include::resources/spring-boot/reactive-examples/example-sse-application.java[]
+----
+
+. Create a Server-sent Event controller class for your application.
+In this example, the class generates a stream of random integers and outputs them into a terminal application.
```

Comment:
_Send_ is IMHO a good equivalent. Using _outputs_ as a 3rd p. verb sound like developer-ese :D Changed.

Reply:
If this means what I think it means (that is, a cmdline application that prints messages to its standard output), then I'd recommend using "prints" instead of "sends".

/me hides